src/rooms.js

```diff
@@ -33,9 +33,19 @@ function filterRoomsByType(type, rooms) {
   return rooms.filter(room => room.roomType === type);
 }
 
+function addBooking(customer, room, date) {
+  const newBooking = {
```

Comment:
You could just return the object instead of creating a variable, but I know that you prefer to do it this way.  Just trying to do an honest job of looking at your code here, haha.